### ISSUE TITLE
Can't open cbr files

### ISSUE THREAD

**Issue by Ospeleo** [MaddTheSane/Simple-Comic#24]
Hi,
I just downloaded Simple Comic via AppStore (MacOS 10.12.6).

I downloaded one cbr file and one cbz file.
I can't open/view neither files via Simple Comic.
The quickview via hitting spacebar from the finder works for the cbz, not the cbr. Is it a sign that Simple Comic is partially working?
When I use SuffitExpander on the cbr file, I get a folder with all the images, so I imagine the cbr file is valid.

Any recommendation ?
thanks

**Comment by MaddTheSane** [comment]
My guess is something else is taking over cbr files.

run `mdls` on the file in question and post the result here.

**Comment by Ospeleo** [comment]
Thanks. 
- I already set .cbr files to open with Simple-Comic
- Simple-Comic do not even open the file using "File>Open" within the application
no luck.

mdls comic.cbr:
_kMDItemOwnerUserID            = 501
kMDItemContentCreationDate     = 2020-05-04 19:16:51 +0000
kMDItemContentModificationDate = 2020-05-04 19:29:40 +0000
kMDItemContentType             = "cx.c3.cbr-archive"
kMDItemContentTypeTree         = (
    "com.rarlab.rar-archive",
    "public.item",
    "public.data",
    "public.archive",
    "cx.c3.cbr-archive"
)
kMDItemDateAdded               = 2020-05-05 23:22:12 +0000
kMDItemDisplayName             = "comic.cbr"
kMDItemFSContentChangeDate     = 2020-05-04 19:29:40 +0000
kMDItemFSCreationDate          = 2020-05-04 19:16:51 +0000
kMDItemFSCreatorCode           = ""
kMDItemFSFinderFlags           = 0
kMDItemFSHasCustomIcon         = (null)
kMDItemFSInvisible             = 0
kMDItemFSIsExtensionHidden     = 0
kMDItemFSIsStationery          = (null)
kMDItemFSLabel                 = 0
kMDItemFSName                  = "comic.cbr"
kMDItemFSNodeCount             = (null)
kMDItemFSOwnerGroupID          = 20
kMDItemFSOwnerUserID           = 501
kMDItemFSSize                  = 150828669
kMDItemFSTypeCode              = ""
kMDItemKind                    = "Comic Rar"
kMDItemLastUsedDate            = 2020-05-05 23:22:20 +0000
kMDItemLogicalSize             = 150828669
kMDItemPhysicalSize            = 150831104
kMDItemUseCount                = 1
kMDItemUsedDates               = (
    "2020-05-05 22:00:00 +0000"
)

**Comment by gelpke** [comment]
Same problem on an iMac (2011) with High Sierra. Cannot open any comic file. All files are "greyed out". I really hoped this version would function well on High Sierra, since 1.7 isn't stable on High Sierra.

**Comment by Ospeleo** [comment]
I don't want to shoot myself in the foot, maybe Simple-Comic 1.8 is just not compatible with High Sierra (10.12) at all? we are dinosaurs.

**Comment by ThrillD** [comment]
I'm having the same problem, except I'm on Catalina. I have 1.7 and it doesn't work. I was all set to update, but now I'm wondering if it's worth updating at all.

**Comment by 2V3EvG4LMJFdRe** [comment]
Same, nothing opens.

**Comment by ThrillD** [comment]
Well, I took a chance and rebooted a few times. The update seems to work now.

**Comment by 2V3EvG4LMJFdRe** [comment]
Nah, it's not working for me on High Sierra after multiple reboots, removal of settings, reinstall.

**Comment by nickv2002** [comment]
There's a few different, but possibly related issues reported in this thread. If you all could try the latest 1.8.8 version of Simple Comic from the [App Store](https://apps.apple.com/us/app/simple-comic/id1497435571?mt=12), that would be a good place to start. If you still have problems please post a small example file here so we can test it out. Thanks.

**Comment by 2V3EvG4LMJFdRe** [comment]
I'm not sure what I could provide, nothing happens at all. pdf, .cbr, .cbz., anything. Don't open, don't produce an error either. The only thing about is if I try to Quit it, it never responds to it, and I have to manually Force Quit (even though the app itself hasn't crashed and thus doesn't offer the option).

**Comment by koldomi** [comment]
Ver. 1.9.0 not work on MacOS Mojave (10.14.6)... previous versión work very well ;(

**Comment by weekender23** [comment]
> Ver. 1.9.0 not work on MacOS Mojave (10.14.6)... previous versión work very well ;(

having the same problem also. Went back to previous version and was able to open .cbr, .cbz files.

**Comment by nickv2002** [comment]
I hit some problems with a reputable Xcode crash that are preventing me from uploading a build with fixes for then Mojave issue (which was fixed by @MaddTheSane in another issue thread). Work work more on this over the winter break.
